### PR TITLE
Various minor fixes to LHC Linux CMake

### PR DESCRIPTION
**Pull request by jasonsandlin**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -4,60 +4,6 @@
 * text=auto
```
Comment (SebastianPD) [inline .gitattributes:5]: you'll need to change the yml file in Utilities\Pipelines\Tasks. The pipeline calls the bash script and you changed their location

**Review by SebastianPD — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (jasonsandlin) [reply .gitattributes:5]: fixed

File: Build/libHttpClient.Linux/CMakeLists.txt [hunk 2]
```diff
@@ -1,11 +1,14 @@
 cmake_minimum_required(VERSION 3.6)
 
-get_filename_component(PATH_TO_ROOT "../../../.." ABSOLUTE)
+get_filename_component(PATH_TO_ROOT "../../.." ABSOLUTE)
```
[the remark this answers is not in the document]
Comment (jasonsandlin) [reply Build/libHttpClient.Linux/CMakeLists.txt:3]: fixed

File: Build/libHttpClient.Linux/curl_Linux.bash [hunk 3]
```diff
@@ -45,5 +38,4 @@ make
 mkdir -p "$SCRIPT_DIR"/../../Binaries/"$CONFIGURATION"/x64/libcurl.Linux
 cp -R "$PWD"/lib/.libs/* "$SCRIPT_DIR"/../../Binaries/"$CONFIGURATION"/x64/libcurl.Linux
 
-make clean
```
[the remark this answers is not in the document]
Comment (jasonsandlin) [reply Build/libHttpClient.Linux/curl_Linux.bash:48]: reverted this change.  trying to make it go quick when rebuilding but ya its needed

**Review by SebastianPD — APPROVED** [review]

**Pull request merged**